### DISCUSSION TITLE
如何添加 WebView2 控件

### DISCUSSION THREAD

**Discussion by AISophon** [Meloong-Git/PCL#4266] [提问] [answered]
我试图在自定义主页中嵌入一个网页，使用`WebBrowser`报错`对象不支持iscomponentinstalled`；使用`WebView2`时报错`无法创建未知类型" http:/schemas.microsoft.com/winfx/2006/xaml/presentation}WebView2"。”，行号为”1”，行位置为“199”`；无法提供<Windows>控件添加一些引用。因此实在是没有办法，我也是个小白，求各位大佬指导

**Comment by LTCatt** [comment, marked as answer]
WPF 不支持在半透明窗口上添加网页控件，有引用也没救（

**Reply by AISophon** [reply]
好吧
